godtools/Views/TractElements/TractHeaderProperties.swift

```diff
@@ -18,6 +18,11 @@ class TractHeaderProperties: TractProperties {
         self.properties = ["backgroundColor"]
     }
     
+    override func setupDefaultProperties() {
+        self.textScale = 3
```

Comment:
@pablomarti the header actually is not scaled by default. it should be the same size as default text (18). it's the header element in a hero that should be 30 by default.